actions.yaml

```diff
@@ -8,4 +8,10 @@ add-admin-user:
       type: string
       description: User password.
   required: [email, password]
+force-https:
+  description: Configure Discourse to use https. Use lowercase true/false word values.
+  params:
+    force_bool:
+      type: string
+      default: "true"
```

Comment:
I'm not sure we should be doing that via an action. Ideally we'd do this based on how the ingress is configured... If that's not going to be possible for now we may consider another approach though. I think the ingress relation passes has the end URL? Could we use that to confirm if we should force https rather then requiring it to be an action?

Reply:
The ingress relation url will always be HTTP, also we will drop the ingress relation very soon.

```python
            event.relation.data[self.model.app]["url"] = f"http://{name}:{port}/"
```

I agree this shouldn't be a action, maybe this should be a configuration?

Reply:
Adding a new config option to enable this sounds good to me.

Reply:
A new config `force_https` has been added and the `force-https` action was removed, thanks!